_api-reference/cluster-settings.md

```diff
@@ -26,30 +45,64 @@ PUT _cluster/settings
+search.max_buckets | Integer | The maximum number of aggregation buckets allowed in a single response. Default is `65536`.
+snapshot.max_concurrent_operations | Integer | The maximum number of concurrent snapshot operations. Default is `1000`.
+slm.health.failed_snapshot_warn_threshold | String | The number of failed invocations since the last successful snapshot that will indicate a problem as per the health API profile. Default is five repeated failures: `5L`.
+indices.breaker.total.limit | String | The starting limit for the overall parent breaker. Default is `70%` of JVM heap if `indices.breaker.total.use_real_memory` is set to `false`. Default is `95%` of JVM heap if `indices.breaker.total.use_real_memory` is set to `true`. 
+indices.breaker.fielddata.limit | String | The limit for the fielddata breaker. Default is `40%` of JVM heap.
+indices.breaker.fielddata.overhead | Floating point | The constant that all fielddata estimations are multiplied by to determine a final estimation. Default is `1.03`.
+indices.breaker.request.limit | String | The limit for the request breaker. Default is `60%` of JVM heap.
+indices.breaker.request.overhead | Integer | The constant that all request estimations are multiplied by to determine a final estimation. Default is `1`.
+network.breaker.inflight_requests.limit | String | The limit for the in-flight requests breaker. Default is `100%` of JVM heap.
+network.breaker.inflight_requests.overhead | Integer/Time unit | The constant that all in-flight requests estimations are multiplied by to determine a final estimation. Default is `2`.
+script.max_compilations_rate | String | The limit for the number of unique dynamic scripts within a defined interval that are allowed to be compiled. Default is 150 every 5 minutes: `150/5m`.
+cluster.routing.allocation.enable | String | Enable or disable allocation for specific kinds of shards: <br /> <br /> `all` - Allows shard allocation for all types of shards. <br /> <br /> `primaries` - Allows shard allocation for primary shards only. <br /> <br /> `new_primaries` - Allows shard allocation for primary shards for new indexes only. <br /> <br /> `none` - No shard allocations are allowed for any indexes. <br /> <br /> Default is `all`.
+cluster.routing.allocation.node_concurrent_incoming_recoveries | Integer | Configures how many concurrent incoming shard recoveries are allowed to happen on a node. Default is `2`.
+cluster.routing.allocation.node_concurrent_outgoing_recoveries | Integer | Configures how many concurrent outgoing shard recoveries are allowed to happen on a node. Default is `2`.
+cluster.routing.allocation.node_concurrent_recoveries | String | Used to set `cluster.routing.allocation.node_concurrent_incoming_recoveries` and `cluster.routing.allocation.node_concurrent_outgoing_recoveries.` to the same value.
```

Comment:
Is the period at the end of `cluster.routing.allocation.node_concurrent_outgoing_recoveries.` intentional?